src/OpenTelemetry.Instrumentation.MassTransit/OpenTelemetry.Instrumentation.MassTransit.csproj

```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
is there benefit in keeping net462 target instead of just netstandard2.0?

Reply:
I have checked whole code, there is no conditional compilation, so I do not see any reason to do this.
I suppose that net461 was here because net standard 2.0 is not fully compatible with this version. Ref: https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0